Makefile

```diff
@@ -1,8 +1,23 @@
+###########################
+# Configuration Variables #
+###########################
 ORG := github.com/operator-framework
```

Comment:
can we grab this from `git` or `go.mod`?

Reply:
I'm down to add it, would probably make the most sense from the Go mod. What sort of value does this open up for us down the line?